SPECS-EXTENDED/perl-Net-IP/perl-Net-IP.spec

```diff
@@ -0,0 +1,297 @@
+#TODO: BR:/R: perl(IP::Authority) when available
+Summary:        Perl module for manipulation of IPv4 and IPv6 addresses
+Name:           perl-Net-IP
+Version:        1.26
+Release:        32%{?dist}
+# Some ambiguity here, see http://rt.cpan.org/Ticket/Display.html?id=28689
+# HPND (MIT-like) for the IP.pm itself, and "like Perl itself" for all the other
+# scripts included.
+License:        HPND AND (GPL-1.0-or-later OR Artistic-1.0-Perl)
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/Net-IP
+Source:         https://cpan.metacpan.org/modules/by-module/Net/Net-IP-%{version}.tar.gz
```

Comment:
let go with `%{name}-%{version}.ext`

Reply:
Source url has keyword(perl-Net-IP) different from %{name} (NET-IP). Hence, leaving as it is.